server/lib/models/assets.py

```diff
@@ -127,3 +127,46 @@ def transfer_asset(bigchain, source, to, asset_id, sk):
     bigchain.validate_transaction(asset_transfer_signed)
     bigchain.write_transaction(asset_transfer_signed)
     return asset_transfer_signed
+
+
+def escrow_asset(bigchain, source, to, asset_id, sk):
+    asset = bigchain.get_transaction(asset_id['txid'])
+    # Create escrow template with the execute and abort address
+    asset_escrow = bigchain.create_transaction(source, [source, to], asset_id, 'TRANSFER',
+                                               payload=asset['transaction']['data']['payload'])
+
+    # Set expiry time (100 secs from now)
+    time_sleep = 100
+    time_expire = str(float(bigchaindb.util.timestamp()) + time_sleep)
```

Comment:
yeah, there is an issue to store the date in bigchaindb as an int. which makes sense